docs/README.md

```diff
@@ -212,6 +213,22 @@ __Note:__ The default option is `never`.
 
 > :warning: Due to a ["won't fix" bug in the Kubernetes API](https://github.com/kubernetes/kubernetes/issues/41743), the LogDNA agent collects events from the entire cluster, including multiple nodes. To prevent duplicate logs when running multiple pods, the LogDNA agent pods defer responsibilty of capturing events to the oldest pod in the cluster. If that pod is down, the next oldest LogDNA agent pod will take over responsibility and continue from where the previous pod left off.
 
+### Configuring regex for redaction and exclusion/inclusion
+
+You can define rules, using **regex** (regular expressions), to control what log data is ingested:
+ * include *only* specific log lines (with LOGDNA_LINE_INCLUSION_REGEX)
```

Comment:
Maybe we can use backticks for env var names, wdyt? like `LOGDNA_LINE_INCLUSION_REGEX`

Reply:
yep, thanks, done.